### ISSUE TITLE
Consider allowing contributions to "linting" schemas

### ISSUE THREAD

**Issue by handrews** [OAI/OpenAPI-Specification#4141]
We have previously turned away quite a few suggested schema changes, particularly from @karenetheridge and @jeremyfiel, that attempted to enforce sensible but not mandatory constraints.  These often involve SHOULDs or statements that something will be ignored rather than treated as an error.

We've discussed allowing additional schemas that would `$ref` the validation schemas and add these additional constraints.  Let's make a formal decision whether to do that or not, and decide what would be allowed in such schemas so that folks who want to write them can do so with confidence.

**Comment by lornajane** [comment]
Agreed that we should keep to schemas that reflect the spec exactly, there are linting tools that can support adding additional rules if users need something more restrictive. We should update the Contributing file to reflect this.

**Comment by baywet** [comment]
Recap from the last time we talked about it: we should update the contributing file about that. And @handrews volunteered to do it :)
